qa/suites/rgw/multisite/tasks/test_multi.yaml

```diff
@@ -24,3 +24,4 @@ tasks:
 - rgw-multisite-tests:
     config:
       reconfigure_delay: 60
+    args: ['-a', '!bucket_sync_disable'] # filter out disable/enable tests
```

Comment:
strange, this isn't working even though `nosetests -a '!bucket_sync_disable'` works fine on the command line:
```
2020-11-12T22:33:05.871 INFO:tasks.rgw_multisite_tests:running rgw multisite tests on 'tasks.rgw_multi.tests' with args=['-a', '!bucket_sync_disable']
2020-11-12T22:33:05.873 ERROR:teuthology.run_tasks:Saw exception from tasks.
Traceback (most recent call last):
  File "/usr/lib/python3.6/optparse.py", line 1387, in parse_args
    stop = self._process_args(largs, rargs, values)
  File "/usr/lib/python3.6/optparse.py", line 1431, in _process_args
    self._process_short_opts(rargs, values)
  File "/usr/lib/python3.6/optparse.py", line 1513, in _process_short_opts
    raise BadOptionError(opt)
optparse.BadOptionError: no such option: -a
```
https://nose.readthedocs.io/en/latest/usage.html#options

Reply:
turns out the -a option is provided by the `nose.plugins.attrib` plugin, which wasn't being loaded by default. added a new commit to load the default plugins